src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -2130,92 +2131,97 @@ class CreateTableStatementSegment(ansi.CreateTableStatementSegment):
         Ref("IfNotExistsGrammar", optional=True),
         Ref("TableReferenceSegment"),
         # Columns and comment syntax:
-        Sequence(
-            Bracketed(
-                Delimited(
-                    Sequence(
-                        OneOf(
-                            Ref("TableConstraintSegment"),
-                            Ref("ColumnDefinitionSegment"),
-                            Ref("SingleIdentifierGrammar"),
+        AnySetOf(
```

Comment:
This is actually a tiny change. Everything beneath this is indentation

Reply:
This is where the hide whitespace review helps!

<img width="397" alt="image" src="https://user-images.githubusercontent.com/10931297/161783275-ea803fac-36c2-434a-a07f-fb0e7a06822a.png">